kyuubi-server/src/main/scala/org/apache/kyuubi/util/FileUtil.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.util
+
+import java.io.{File, InputStream}
+import java.nio.file.{Files => JFiles}
+import java.nio.file.Path
+
+import scala.util.Random
+
+import com.google.common.io.{Files => GFiles}
+
+object FileUtil {
+
+  /**
+   * write InputStream to temp file in directory
+   */
+  def writeToTempFile(is: InputStream, dir: Path, fileName: String): File = {
+    val prefix = GFiles.getNameWithoutExtension(fileName)
+    val suffix = GFiles.getFileExtension(fileName)
+    val file = new File(dir.toFile, s"$prefix${Random.nextLong()}.$suffix")
+    file.deleteOnExit()
+    JFiles.copy(is, file.toPath);
```

Comment:
Changed to use Java's `Files` here, without using `commons-io`'s `FileUtils`. @turboFei